aptos-move/framework/supra-framework/sources/automation_registry_state.move

```diff
@@ -0,0 +1,226 @@
+/// Supra Automation Registry State
+///
+/// This contract is part of the Supra Framework and is designed to manage automated task entries
+module supra_framework::automation_registry_state {
+
+    use std::signer;
+    use std::vector;
+    use supra_framework::event;
+
+    use supra_std::enumerable_map::{Self, EnumerableMap};
+
+    use supra_framework::system_addresses;
+    use supra_framework::timestamp;
+
+    friend supra_framework::genesis;
```

Comment:
We don't need to add the `genesis` module as a friend in the `automation_registry_state` module.